stereo_points/stereo_points.py

```diff
@@ -229,70 +227,61 @@ def onControlPointSelectedChanged(self, newNode):
-                                                                         updatedNode.GetID())
-        fiducialNode = slicer.mrmlScene.GetNodeByID(matchingControlPoint_nodeID)
+        fiducialNode = slicer.mrmlScene.GetNodeByID(updatedNode.GetNodeReferenceID("vtkMRMLMarkupsLineNode"))
         fiducialNode.SetName(updatedNode.GetName().replace('_coordsConversion', ''))
 
     def onControlPointNodeModified(self, updatedNode, eventType):
         logging.debug("enter onControlPointNodeModified")
-        matchingCoordTable_nodeID = self.findMatchingNodeInIdTupleList(self.fiducialNodesToCoordTables_id_tuples,
-                                                                       updatedNode.GetID())
-        coordTable = slicer.mrmlScene.GetNodeByID(matchingCoordTable_nodeID)
+        coordTable = slicer.mrmlScene.GetNodeByID(updatedNode.GetNodeReferenceID("vtkMRMLTableNode"))
         coordTable.SetName(updatedNode.GetName() + '_coordsConversion')
         self.updatePointsAfterMove(coordTable, updatedNode)
 
-    def findMatchingNodeInIdTupleList(self, tupleList, search):
-        # print('looking for '+str(search)+ ' in '+str(tupleList))
-        return [j for j in [i for i in tupleList if search in i][0] if j != search][0]
-
     def onAddBtnClicked(self):
+        coordTable = slicer.mrmlScene.GetNodeByID(self.fiducialGroup_selectionCombo.currentNode().GetNodeReferenceID("vtkMRMLTableNode"))
 
-        coordTable = slicer.mrmlScene.GetNodeByID(
-            self.findMatchingNodeInIdTupleList(self.fiducialNodesToCoordTables_id_tuples,
-                                               self.fiducialGroup_selectionCombo.currentNode().GetID()))
         # print('[onAddBtnClicked] x: %f | y: %f | z: %f | r: %f | a: %f'%(
         # self.xField.value, self.yField.value, self.zField.value,
         # self.ringField.value, self.arcField.value))
         self.addPointFromStereoSetting(coordTable,
                                        self.xField.value, self.yField.value, self.zField.value,
                                        self.ringField.value, self.arcField.value, self.depthField.value,
+                                       slicer.mrmlScene.GetNodeByID(self.fiducialGroup_selectionCombo.currentNodeID).GetName() + 
+                                       "_" + 
                                        self.nameField.text
                                        )
         self.table2ControlPoint(coordTable, self.fiducialGroup_selectionCombo.currentNode())
 
         self.nameField.setText('')
-        self.xField.setValue(0.0)
-        self.yField.setValue(0.0)
-        self.zField.setValue(0.0)
-        self.ringField.setValue(0.0)
-        self.arcField.setValue(0.0)
-        self.depthField.setValue(0.0)
+        # self.xField.setValue(0.0)
+        # self.yField.setValue(0.0)
+        # self.zField.setValue(0.0)
+        # self.ringField.setValue(0.0)
+        # self.arcField.setValue(0.0)
+        # self.depthField.setValue(0.0)
```

Comment:
Stylewise we don't like commented blocks of code like this ("dead code") unless it's there's a clear comment about why keeping it around might be valuable to developers in the future.  Better to get rid of this code if it's really not needed anymore.